bftengine/include/bftengine/ControlStateManager.hpp

```diff
@@ -14,19 +14,47 @@
 #pragma once
 #include <optional>
 #include "IStateTransfer.hpp"
+#include "ReservedPages.hpp"
+#include "Serializable.h"
 
 namespace bftEngine {
 
-class ControlStateManager {
+class ControlStateManager : public ResPagesClient<ControlStateManager, 1, 1> {
```

Comment:
Nit: Without looking at the definition of `ResPagesClient`, I have no idea what these parameters mean. Maybe create some named constexprs for the `1s` to make it a bit clearer to the reader.

Reply:
fixed